kotest-framework/kotest-framework-engine/src/jvmTest/kotlin/com/sksamuel/kotest/engine/spec/isolation/perleaf/FreeSpecInstancePerLeafTest.kt

```diff
@@ -19,7 +19,7 @@ class FreeSpecInstancePerLeafTest : FreeSpec() {
 
    init {
 
-      finalizeSpec {
+      afterProject {
```

Comment:
Feels a bit weird to me to translate a `afterSpec` to be a `afterProject`. Are these cases where `afterSpec` isn't enough?

Reply:
Yes it's checking after all the specs have run what the total count is.